cosmwasm/verify

```diff
@@ -29,28 +50,105 @@ while getopts ':hn:' option; do
+  echo "$usage"
+  exit 1
+fi
+
+if [[ ! -z $code_id && ! -z $contract_addr ]]; then
+  printf "Both of the -i and -a parameters cannot be specified.\n"
+  echo "$usage"
+  exit 1
+fi
+
+if [[ -z $chain || -z $network || -z $wasm ]]; then
+  printf "The -c, -n, and -w parameters are required.\n"
+  echo "$usage"
+  exit 1
+fi
 
+url=""
+jq_cmd=""
+code_id_url=""
+code_id_jq=""
+# Fill in the above values based on network and chain
 case "$network" in
-  mainnet) url="https://lcd.terra.dev";;
-  testnet) url="https://bombay-lcd.terra.dev";;
-  devnet)  url="http://localhost:1317";;
+  mainnet) 
+    case "$chain" in
+      terra) url="https://columbus-lcd.terra.dev/terra/wasm/v1beta1/codes/"
+        jq_cmd="jq '.code_info.code_hash' -r | base64 -d | hexdump -v -e '/1 \"%02x\" '"
+        code_id_url="https://columbus-lcd.terra.dev/terra/wasm/v1beta1/contracts/"
+        code_id_jq="jq '.contract_info.code_id' -r "
+        ;;
+      terra2) url="https://phoenix-lcd.terra.dev/wasm/code/"
+        jq_cmd="jq '.result.data_hash' -r "
+        code_id_url="https://phoenix-lcd.terra.dev/wasm/contract/"
+        code_id_jq="jq '.result.code_id' -r "
+        ;;
+      xpla) url="https://dimension-lcd.xpla.dev/cosmwasm/wasm/v1/code/"
+        jq_cmd="jq '.code_info.data_hash' -r "
+        code_id_url="https://dimension-lcd.xpla.dev/cosmwasm/wasm/v1/contract/"
+        code_id_jq="jq '.contract_info.code_id' -r"
+        ;;
+    esac ;;
+  testnet) 
+    case "$chain" in
+      terra2) url="https://pisco-lcd.terra.dev/wasm/code/"
+        jq_cmd="jq '.result.data_hash' -r "
+        code_id_url="https://pisco-lcd.terra.dev/wasm/contract/"
+        code_id_jq="jq '.result.code_id' -r "
+        ;;
+      injective) url="https://k8s.testnet.exchange.grpc-web.injective.network/api/explorer/v1/wasm/codes/"
+        jq_cmd="jq '.checksum.hash' -r | cut -d 'x' -f2"
+        code_id_url="https://k8s.testnet.lcd.injective.network/cosmwasm/wasm/v1/contract/"
+        code_id_jq="jq '.contract_info.code_id' -r"
+        ;;
+      xpla) url="https://cube-lcd.xpla.dev/cosmwasm/wasm/v1/code/"
+        jq_cmd="jq '.code_info.data_hash' -r "
+        code_id_url="https://cube-lcd.xpla.dev/cosmwasm/wasm/v1/contract/"
+        code_id_jq="jq '.contract_info.code_id' -r "
+        ;;
+    esac ;;
+  devnet)  
+    case "$chain" in
+      terra) url="http://localhost:1317/terra/wasm/v1beta1/codes/"
+        jq_cmd="jq '.code_info.code_hash' -r | base64 -d | hexdump -v -e '/1 \"%02x\" '"
+        ;;
+      terra2) url="http://localhost:1318/terra/wasm/v1beta1/codes/"
+        jq_cmd="jq '.code_info.code_hash' -r | base64 -d | hexdump -v -e '/1 \"%02x\" '"
+        ;;
+    esac ;;
   *) printf "Network not set. Specify with -n\n" >&2
      echo "$usage" >&2
      exit 1
      ;;
 esac
 
-[ $# -ne 2 ] && { echo "$usage" >&2; exit 1; }
-obj_file=$1
-code_id=$2
+if [[ -z $url ]]; then
+  printf "The combination of $network and $chain is not supported.\n"
```

Comment:
When printing errors, make sure to print them to stderr and not stdout
```suggestion
  printf "The combination of $network and $chain is not supported.\n" 1>&2 
```

Reply:
Fixed